module3/Homework3Playground.playground/Contents.swift

```diff
@@ -0,0 +1,175 @@
+
+
+//a) Create an array of Int called nums with values 0 through 20. Iterate over the Array and print the even numbers.
+print("a) ---------------")
+let firstArry = Array(0...20)
+for num in firstArry where num % 2 == 0 {
+  print(num)
+}
+
+//b) In your Playground, define the following String:
+//
+//let sentence = “The qUIck bRown fOx jumpEd over the lAzy doG”
+//
+//Iterate over sentence counting the vowels (a, e, i, o, u), ignoring the case.
+
+print("b) ---------------")
+let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"
+var vowelCount = 0
+let lowercasedSentence = sentence.lowercased()
+let vowels = Set(["a", "e", "i", "o", "u"])
+for character in lowercasedSentence where vowels.contains(String(character)) {
+  vowelCount += 1
+}
+print(vowelCount)
+
+
+//c) Create two arrays of Int with the values of 0 through 4. Use a nested for loop to print a mini multiplication table. The output, which should be multiple lines, should be in the following format:
+//
+//0 * 0 = 0
+//
+//
+//The format follows the number from the first array, space, followed by the character *, space,  followed by the number from the second array, space,  followed by =, space, followed by the result of the number from the first array multiplied by the number from the second array (just like the example above). There are a couple of ways to achieve this, but String Interpolation is the easiest.
+
+print("c) ---------------")
+let arrayA = Array(1...4)
```

Comment:
These should be ranges of 0...4 to match the requirement. Your code would omit the "0 * 0 = 0" row.

Reply:
Good catch - this was an oversight on my part.